charts/fybrik/values.yaml

```diff
@@ -14,7 +14,11 @@ clusterScoped: true
 
 # The namespace where Fybrik deploys data path components (modules)
 modulesNamespace:
-  # The Fybrik namespace name, if not defined, default "fybrik-blueprints" is used
+# The Fybrik namespace name, if not defined, default "fybrik-blueprints" is used.
+# If `app.fybrik.io/modules-namespace` label is set in FybrikApplication resource then
+# it takes precedence over other values defined. In this case, it is assumed that Fybrik
+# is not responsible to create this namespace and remove it when Fybrik is uninstalled.
+# Moreover, it is a user responsibility to give Fybrik the permissions to deploy modules in this namespace.
```

Comment:
This assumption contradicts the deployment value (`managedByFybrik`)
If Fybrik manages the namespace then the application label should be ignored.

Reply:
No, the managed by fybrik is a flag relevant only to the specific namespace defined in` .Values`. The application labels have priority over `.Values` settings

Reply:
I added a comment about that to the `managedByFybrik` field.